src/main/kotlin/no/nav/familie/ba/mottak/util/IdentUtils.kt

```diff
@@ -8,4 +8,20 @@ fun erDnummer(personIdent: PersonIdent): Boolean {
 
 fun erDnummer(personIdent: String): Boolean {
     return personIdent.substring(0, 1).toInt() > 3
+}
+
+fun erFDatnummer(personIdent: String): Boolean {
+    return personIdent.substring(6)?.toInt()!! == 0
+}
+
+fun erFDatnummer(personIdent: PersonIdent): Boolean {
+    return if (personIdent.id == null) false else erFDatnummer(personIdent.id!!)
+}
+
+fun erBostNummer(personIdent: String): Boolean {
+    return personIdent.substring(1, 2)?.toInt()!! > 1
```

Comment:
Blir dette riktig da? Her plukkes 2. siffer i rekken ut mens det vel var 3. siffer som skulle sjekkes?
Det finnes en testklasse for IdentUtils som evt kan utvides med noen enkle tester for dette.

Reply:
fikser